azure-pipelines.yml

```diff
@@ -407,142 +407,131 @@ jobs:
 
   pool:
     vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+      Python38:
+        python.version: '3.8'
   steps:
-  - task: UsePythonVersion@0
-    displayName: 'Use Python 3.8'
-    inputs:
-      versionSpec: 3.8
-  - task: DownloadPipelineArtifact@1
-    displayName: 'Download Python Wheels'
-    inputs:
-      TargetPath: '$(Build.ArtifactStagingDirectory)/pypi'
-      artifactName: pypi
-  - bash: |
-      pip install --force-reinstall wheel
-  - task: Bash@3
-    displayName: 'Run Test Automation'
-    inputs:
-      targetType: 'filePath'
-      filePath: ./scripts/ci/test_automation.sh
-    env:
-      ADO_PULL_REQUEST_TARGET_BRANCH: $(System.PullRequest.TargetBranch)
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '$(python.version)'
+      displayName: 'Use Python $(python.version)'
+    - bash: |
+        set -ev
+
+        git fetch origin dev
+
+        python -m venv env
+        chmod +x env/bin/activate
+        source env/bin/activate
 
-- job: RunAutomationReduced20190301Python3_8
-  displayName: Run Automation Reduced Python 3.8, Profile 2019-03-01
+        pip install -q azdev
+        azdev --version
+
+        azdev setup -c ./
+      displayName: Setup Env
+    - bash: |
+        set -ev
+
+        source env/bin/activate
+
+        azdev test --series --profile 2019-03-01-hybrid
+      displayName: "Automation Test"
+
+- job: AutomationTest20180301
+  displayName: Automation Test (Profile 2018-03-01)
   dependsOn: BuildPythonWheel
+  timeoutInMinutes: 30
 
   pool:
     vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+      Python38:
+        python.version: '3.8'
   steps:
-  - task: UsePythonVersion@0
-    displayName: 'Use Python 3.8'
-    inputs:
-      versionSpec: 3.8
-  - task: DownloadPipelineArtifact@1
-    displayName: 'Download Python Wheels'
-    inputs:
-      TargetPath: '$(Build.ArtifactStagingDirectory)/pypi'
-      artifactName: pypi
-  - bash: |
-      pip install --force-reinstall wheel
-  - task: Bash@3
-    displayName: 'Run Test Automation'
-    inputs:
-      targetType: 'filePath'
-      filePath: ./scripts/ci/test_automation.sh
-    env:
-      REDUCE_SDK: 'True'
-      AZURE_CLI_TEST_TARGET_PROFILE: '2019-03-01'
-      ADO_PULL_REQUEST_TARGET_BRANCH: $(System.PullRequest.TargetBranch)
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '$(python.version)'
+      displayName: 'Use Python $(python.version)'
+    - bash: |
+        set -ev
+
+        git fetch origin dev
+
+        python -m venv env
+        chmod +x env/bin/activate
+        source env/bin/activate
+
+        pip install -q azdev
+        azdev --version
+
+        azdev setup -c ./
+      displayName: Setup Env
+    - bash: |
+        set -ev
+
+        source env/bin/activate
+
+        azdev test --series --profile 2018-03-01-hybrid
+      displayName: "Automation Test"
```

Comment:
for 2018-03-01-hybrid?

Reply:
You mean suggesting change the `displayName` value to `Automation Test for 2018-03-01-hybrid` ?

Reply:
Changed.